File: app/src/main/java/com/github/krot/utils/PropertiesLoader.java
package com.github.krot.utils;

import com.github.krot.sqlite.helper.DatabaseHelper;

public class PropertiesLoader {

    public static void loadSystemProperties() {
        DatabaseHelper databaseHelper = DatabaseHelper.getHelper();

        // Next round
        String value = databaseHelper.getProperty(Properties.NEXT_ROUND.toString());
        if (value != null) {
            SystemProperties.NEXT_ROUND = Integer.valueOf(value);
        }
    }
}

File: app/src/main/kotlin/com/github/krot/utils/PropertiesLoader.kt
package com.github.krot.utils

import com.github.krot.sqlite.helper.DatabaseHelper

class PropertiesLoader {
    companion object {
        @JvmStatic fun loadSystemProperties() {
            val databaseHelper = DatabaseHelper.INSTANCE

            // Next round
            val value = databaseHelper.getProperty(Properties.NEXT_ROUND.toString())
            if (value != null) {
                SystemProperties.NEXT_ROUND = value.toInt()
            }
        }
    }
}
